### ISSUE TITLE
Is there any reason why AltStore doesn't support iPad Pro (12.9) screen resolution?

### ISSUE THREAD

**Issue by rtxbae** [rileytestut/AltStore#1031]
I was surprised to see that it doesn't support iPad Pro 12.9 screen resolution since it's an actively developed app. Is there any technical reason why it's not supported? Here's the app in 12.9 iPad Pro:

![image](https://user-images.githubusercontent.com/65499097/177022351-ea84475d-2dcb-476a-8c48-dd27e8170269.jpeg)

**Comment by IOplus** [comment]
I don't think it supports any iPad, I'm using the iPad air 4 and it's the same. Would be nice though...
